storage/client/queries/queries.go

```diff
@@ -464,7 +464,7 @@ const (
 				(SELECT sum(amount) from chain.runtime_transfers where runtime = $1 AND receiver = $2::text)
 				, 0) AS total_received,
 			COALESCE (
-				(SELECT count(*) from chain.runtime_related_transactions where runtime = $1 AND account_address = $2::text)
+				(SELECT num_txs from stats.runtime_accounts where runtime = $1 AND address = $2::text)
 				, 0) AS num_txns`
```

Comment:
```suggestion
				, 0) AS num_txs`
```

I'd prefer to standardize this but this would be a breaking change in the api...